### PR TITLE
Added comma dangle rule in src/js

### LINKED ISSUE #154: Add eslint rule comma-dangle: "only-multiline"
See https://eslint.org/docs/rules/comma-dangle . Should _only_ apply to the JS in `src/`.

### PR DESCRIPTION
**Pull request by Yashs911**
Fixes #154

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -46,5 +46,13 @@ module.exports = {
                 "no-var": ["off"],
             }
         },
+        {
+            "files": [
+                "src/js/**/*.js"
+            ],
+            "rules": {
+                "comma-dangle": ["error", "only-multiline"]
```
Comment (cdrini) [inline .eslintrc.js:54]: Hmmm, could we add this with the other rules, above? Or is there a reason you created it as a separate section?

Comment (Yashs911) [reply .eslintrc.js:54]: @cdrini in the PR it was mentioned only to JS in src/
If you want I can add this rule with other rules

Comment (cdrini) [reply .eslintrc.js:54]: Let's try adding it to everything :+1:

**Review by cdrini — APPROVED** [review]
Lgtm!

**Pull request merged**